extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -434,7 +494,8 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
 
       const seriesDeliveredPromises = seriesPromises.map(promise =>
         promise.then(instances => {
-          storeInstances(instances);
+          const clientName = seriesClientsMapping[instances[0]['0020000E'].Value[0]];
```

Comment:
Please a const and use it here (like you have done for study uid) SERIES_INSTANCE_UID = '0020000E';